src/run/runner/executor.rs

```diff
@@ -1,11 +1,8 @@
-use super::helpers::env::BASE_INJECTED_ENV;
 use super::interfaces::{ExecutorName, RunData};
 use crate::prelude::*;
 use crate::run::instruments::mongo_tracer::MongoTracer;
 use crate::run::{check_system::SystemInfo, config::Config};
```

Comment:
nit: import grouping

Reply:
we could set this in our repos: https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=granula#Crate%5C%3A

Reply:
nevermind it's unstable and therefore not able to be put in rustfmt.toml, it needs to be set at IDE level :cry: